### DISCUSSION TITLE
Running Prisma 2 rust server separately - eg K8S sidecar container

### DISCUSSION THREAD

**Discussion by johnlister** [prisma/prisma#7321] [Q&A] [answered]
I was wondering if it is possible to split the Prisma 2 rust server from the javascript client, rather than them being bundled together and the server started by the client code. 
An example scenario is to run the server as a full sidecar container within a Kubernetes pod. 

Another possible scenario is to run the rust server as a standalone cluster and point multiple micro services(Prisma client) at the cluster allowing a higher level approach to managing connections and load. I appreciate this seems like a move backwards to the approach taken by Prisma 1.

**Comment by ryands17** [comment, marked as answer]
@johnlister 👋 

I don't think this is possible at the moment and this request discusses the same: https://github.com/prisma/prisma/issues/100

Currently you would need the query engine and client together and yes the approach above would be similar as to what was in P1 via a separate container just for querying the database.

**Reply by johnlister** [reply]
I'd seen the same issue, it would be good to be able to configure if the prisma2 "server" is started automatically and if not where it is running for those use cases where it may be desirable.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
